README.md

```diff
@@ -121,5 +131,6 @@ config :demo, MyAppWeb.Endpoint,
 You can also optionally import the style for the default CSS classes in your `app.css` file.
 
 ```css
+// assets/css/app.css
```

Comment:
Could you change this to use css comment syntax, so the highlighting is correct?

```css
/* my comment */
```

Reply:
sure! 🙌